lib/bolt/bolt_option_parser.rb

```diff
@@ -28,6 +28,9 @@ def get_help_text(subcommand, action = nil)
       when 'file'
         { flags: ACTION_OPTS + %w[tmpdir],
           banner: FILE_HELP }
+      when 'inventory'
+        { flags: OPTIONS[:inventory] + OPTIONS[:global],
```

Comment:
Add `format` as an option. Does it make sense to have all display options available?

Reply:
I think all the display options make sense. 
`--boltdir` and `--configfile` should probably be supported too. I think it makes sense to move them to a separate group from `modulepath`